### PR TITLE
something changed

### PR DESCRIPTION
**Pull request by szekelydavid92**
changed ..

### CODE REVIEW

**Review by amdor — APPROVED** [review]
If that qualifies as a fix, I shall approve it...

File: src/main/java/com/mycompany/app/App.java [hunk 1]
```diff
@@ -9,5 +9,6 @@
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
+        // No bugs found, but I'm going to push and open a pull request.
```
Comment (amdor) [inline src/main/java/com/mycompany/app/App.java]: Almost forgot to suggest changes...

**Review by amdor — CHANGES_REQUESTED** [review]

File: src/main/java/com/mycompany/app/App.java (hunk 1 above)
Comment (szekelydavid92) [reply src/main/java/com/mycompany/app/App.java]: Fixed. If it is good now, please approve my pul request.

**Review by amdor — APPROVED** [review]
Even though....

**Pull request merged**